nodetypeimplementations/https%3A%2F%2Fust-quantil.github.io%2Fnodetypeimplementations/QHAna-PluginRunner-Implementation_latest-w2-wip1/NodeTypeImplementation.tosca

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:ns0="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="https://ust-quantil.github.io/nodetypeimplementations" id="nodeTypeImplementationsQuantil-QHAna-PluginRunner-Implementation_latest-w2-wip1">
+    <NodeTypeImplementation xmlns:nodeTypesQuantil="https://ust-quantil.github.io/nodetypes" targetNamespace="https://ust-quantil.github.io/nodetypeimplementations" name="QHAna-PluginRunner-Implementation_latest-w2-wip1" abstract="no" final="no" nodeType="nodeTypesQuantil:QHAna-PluginRunner_latest-w2-wip1">
```

Comment:
s.a.

However, release this as the last element as you need to update the references to the IA/DAs.